Dangerfile

```diff
@@ -1,19 +1,20 @@
-require 'pmdtester/runner'
+require 'pmdtester'
 require 'time'
 require 'logger'
 
 @logger = Logger.new(STDOUT)
 
 def run_pmdtester
   Dir.chdir('..') do
-    argv = ['-r', './pmd', '-b', "#{ENV['TRAVIS_BRANCH']}", '-p', 'FETCH_HEAD', '-m', 'online']
+    argv = ['-r', './pmd', '-b', "#{ENV['TRAVIS_BRANCH']}", '-p', 'FETCH_HEAD', '-m', 'online', '-a']
     Process.fork do
 	  begin
         runner = PmdTester::Runner.new(argv)
-        runner.run # Optimize: get more information from runner, e.g. introduce new pmd errors?
+        introduce_new_pmd_errors = runner.run
+        warn("The PR may introduce new PMD errors!") if introduce_new_pmd_errors
 	  rescue StandardError => e
-	    @logger.error "Running pmdtester failed: #{e.msg}"
 	    warn("Running pmdtester failed, this message is mainly used to remind the maintainers of PMD.")
+	    @logger.error "Running pmdtester failed: #{e.inspect}"
```

Comment:
I noticed that pmdtester failed in https://github.com/pmd/pmd/pull/1277 
CI message: https://travis-ci.org/pmd/pmd/jobs/410862698#L5509
Danger failed before it issued a warning, hope this will fix the issue.

Reply:
:+1: